app.js

```diff
@@ -25,6 +25,11 @@ if (process.env.NODE_ENV !== 'production') { require('./webpack/webpack')(app);
 
 const hbs = require('hbs');
 
+hbs.registerHelper('clientConfig', () => JSON.stringify({
```

Comment:
This would be cleaner as a generic helper to render json and passing the config to the template as an object. Like that the template can use `{{{ json clientConfig }}}` while still be able to access specific properties on the config object if needed

Reply:
good idea, doin' it